src/app/app-routing.module.ts

```diff
@@ -47,7 +47,7 @@ const routes: Routes = [
 ];
 
 @NgModule({
-  imports: [RouterModule.forRoot(routes, { onSameUrlNavigation: 'reload', scrollPositionRestoration: 'enabled' })],
```

Comment:
Removed because when paginating, the view would reset to the top of the screen. Not sure if this will bring back any problems.